LSP-tailwindcss.sublime-settings

```diff
@@ -68,25 +63,23 @@
 	],
 	// @see https://github.com/tailwindlabs/tailwindcss-intellisense/blob/45c5cd6d4a0bcaafbeabb661e505c86de5e7d120/src/lsp/server.ts#L82-L86
 	"initializationOptions": {
-		// seems related to https://github.com/tailwindlabs/tailwindcss-intellisense#tailwindcssincludelanguages
-		// it looks like a syntax (re-)mapping whose keys/values are old/new languageIds
+		// @see https://github.com/tailwindlabs/tailwindcss-intellisense#tailwindcssincludelanguages
 		"userLanguages": {}
 	},
```

Comment:
As far as I can see this is only handling in the VSCode extension so is not relevant to ST.

It's referenced in the settings schema in the server but only extension does something with it: https://github.com/tailwindlabs/tailwindcss-intellisense/search?q=includeLanguages

Reply:
c8e69dd see the commit description for more info